source/command-monitoring/tests/command.yml

```diff
@@ -0,0 +1,45 @@
+data:
+  - { _id: 1, x: 11 }
+
+collection_name: &collection_name "values"
+database_name: &database_name "command-monitoring-tests"
+
+tests:
+  -
+    description: "A successful command"
+    operation:
+      name: "count"
+      arguments:
+        filter: { _id: 1 }
+    expectations:
+      -
+        command_started_event:
+          command:
+            count: *collection_name
+            query: { _id: 1 }
+          command_name: "count"
+          database_name: *database_name
+      -
+        command_succeeded_event:
+          reply:
+            - { ok: 1, n: 1 }
+          command_name: "count"
+          database_name: *database_name
+  -
+    description: "A failed command event"
+    operation:
+      name: "count"
+      arguments:
+        filter: { $nothing: true }
+    expectations:
+      -
+        command_started_event:
+          command:
+            count: *collection_name
+            query: { $nothing: true }
```

Comment:
I'm assuming this is related to trying to make a query fail. If so, can we call this something else? $illegalOperator or something?